src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -13,12 +13,8 @@
 import uk.gov.hmcts.reform.sscs.ccd.domain.Representative;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
 import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
-import uk.gov.hmcts.reform.sscs.model.single.hearing.IndividualDetails;
-import uk.gov.hmcts.reform.sscs.model.single.hearing.OrganisationDetails;
-import uk.gov.hmcts.reform.sscs.model.single.hearing.PartyDetails;
-import uk.gov.hmcts.reform.sscs.model.single.hearing.RelatedParty;
-import uk.gov.hmcts.reform.sscs.model.single.hearing.UnavailabilityDayOfWeek;
-import uk.gov.hmcts.reform.sscs.model.single.hearing.UnavailabilityRange;
+import uk.gov.hmcts.reform.sscs.model.hmc.reference.PartyType;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.*;
```

Comment:
Not necessary for now but shouldn't ever use * imports

Reply:
yeah it's a setting in intelliJ, we should raise the limit to 99 (that was the old fix anyway lol)